packages/sdk/src/contracts/prebuilt-implementations/marketplace.ts

```diff
@@ -438,6 +438,29 @@ export class Marketplace implements UpdateableNetwork {
     await this.contractWrapper.multiCall(encoded);
   }
 
+  /**
+   * returns the minimum bid a user can place to outbid the previous highest bid
+   * @param listingId - the listing id of the auction
+   */
+  public async getMinimumNextBid(
```

Comment:
`this.auction` is being use on other functions as well in this class. So I assumed, this was the appropriate place to add the function and also since the `getBidBufferBps` is also here. It made it more convenient for me to implement here basically.